modules/add_service.rb

```diff
@@ -0,0 +1,22 @@
+module AddService
+  def add_service(option)
+    case option
+    when 7
+      add_book
+    when 8
+      add_music_album
+    when 9
+      add_game
+    end
+  end
+
+  def add_book
+    puts 'This is for adding a book'
+  end
+
+  def add_music_album; end
+
+  def add_game
```

Comment:
- [ ] This module `add_service` is called nowhere and it has methods `add_book`, `add_music_album` and `add_game` that are already defined in the `books_services`, `music_album_services`, and `games_services` respectively.

Reply:
Hi Selma, I have deleted the add_service module. That was one of the changes I did. I realized if we have one add_service module, it will be quite large that's why I separated it.